src/components/Main/Filter.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import '../../styles/filter.scss';
+
+
+class Filter extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            selectedItem: 'All'
+        };
+
+        this.handleClick = this.handleClick.bind(this);
+    }
+
+    handleClick(item) {
+        this.setState({selectedItem: item});
+    }
+
+    render() {
+        const genres = ['All', 'Documentary', 'Comedy', 'Horror', 'Crime'];
```

Comment:
this const well be re-created with every render but in reality it doesn't use any properties or state. It can be stored somewhere outside the component

Reply:
Fixed, thank you.